model/general/createCalendar.js

```diff
@@ -2,6 +2,36 @@ const client = require('../../database/pg.js');
 const utils = require('../../lib/hashUtils.js');
 
 const createCalendar = (data, callback) => {
+  const cookie = [ data.cookies.jobsite ];
+  if (cookie[0]) {
+    const userIdQueryStr = `SELECT user_id FROM session WHERE cookie=($1)`;
```

Comment:
When creating a string only use ` if it's a string literal (example: `${variable}`), otherwise just use regular ''